dist/style/index.css

```diff
@@ -194,7 +194,7 @@
 
 .buttons {
     display: flex;
-    margin-top: auto;
```

Comment:
Just so that I'm fully aware of what's going on, what prompted this change? ^^

Reply:
we want the tags to be close to the buttons, so tags are now margin-top: auto and buttons have a fixed top margin.

without this the tag position would depend on the blurb height.